.travis.yml

```diff
@@ -25,6 +25,19 @@ deploy:
     app: move.mil
     env: move-mil-staging
     bucket_name: elasticbeanstalk-us-east-1-321940898149
+    bucket_path: move-mil-staging
+    on:
+      branch: master
```

Comment:
Should push ZIP files to the S3 bucket in a `move-mil-staging` folder and specifies `master` as a branch condition (`master` is implied, but adding it here explicitly to avoid confusion).